### ISSUE TITLE
[Blazor, NET5] Tags are sanitized differently from .NET Framework

### ISSUE THREAD

**Issue by vankampenp** [mganss/HtmlSanitizer#298]
When running the following code in the default .NET fiddle with .NET 4.7.2

```
using System;
using Ganss.XSS;


					
public class Program
{
	public static void Main()
	{
		var sanitizer = new HtmlSanitizer();
var html = @"<script>alert('xss')</script><div onload=""alert('xss')"""
    + @"style=""background-color: test"">Test<img src=""test.gif"""
    + @"style=""background-image: url(javascript:alert('xss')); margin: 10px""></div>";
var sanitized = sanitizer.Sanitize(html, "http://www.example.com");
	Console.WriteLine(sanitized);	
	}
}
```
The output is as expected:
`<div style="background-color: test">Test<img src="http://www.example.com/test.gif" style="margin: 10px"></div>`

However, when changing to .NET 5 and HtmlSanitizer 6.0.437 the output is stripped of the style attributes:

`<div>Test<img src="http://www.example.com/test.gif" style="margin: 10px"></div>`

The same happens in my own code in Blazor with .NET 5

**Comment by mganss** [comment]
The .NET fiddle seems to be using an older version (I can't tell which one exactly). The output you're seeing with the latest version of HtmlSanitizer is what's expected (see also https://github.com/AngleSharp/AngleSharp/issues/101). I'll update the fiddle.

**Comment by vankampenp** [comment]
Thx, that clarifies.
What I was trying to do is this:

using System;
using Ganss.XSS;


					
```
public class Program
{
	public static void Main()
	{
		var sanitizer = new HtmlSanitizer();
var html = @"<span style=""font-size: xxx-large;"">Flexibiliteits Index Test (FIT-60)</span>";
var sanitized = sanitizer.Sanitize(html);
	Console.WriteLine(sanitized);	
	}
}
```
Here the font-size is removed. When I change the font-size to xx-large it works. It seems xxx-large is seen as an illegal property

**Comment by mganss** [comment]
It seems this is unsupported by AngleSharp.Css because it's fairly new (e.g. currently [unsupported by Safari and others](https://caniuse.com/mdn-css_properties_font-size_xxx-large), too). I've opened an issue here: https://github.com/AngleSharp/AngleSharp.Css/issues/77

**Comment by vankampenp** [comment]
Thx, it comes from a component I am using. 

For now, I will do a search and replace:

` return sanitizer.Sanitize(html.Replace("xxx-large","48px",StringComparison.CurrentCultureIgnoreCase));`

**Comment by mganss** [comment]
Fixed in 6.0.441 via AngleSharp 0.16.1
